test/handlers/erc20/setResourceIDAndContractAddress.js

```diff
@@ -55,18 +55,46 @@ contract('ERC20Handler - [setResourceIDAndContractAddress]', async () => {
         assert.strictEqual(secondERC20ResourceID.toLowerCase(), retrievedResourceID.toLowerCase());
     });
 
-    it('should revert because resourceID should already be set', async () => {
-        await TruffleAssert.reverts(BridgeInstance.adminSetResource(
-            ERC20HandlerInstance.address, initialResourceIDs[0], ERC20MintableInstance1.address),
-            "resourceID already has a corresponding contract address");
+    it('resourceID should be updated correctly with new token contract address', async () => {
```

Comment:
For this test case i think "existing resourceID should be updated correctly with new token contract address" may be more clear